src/js/components/aboutTheData/AboutTheDataPage.jsx

```diff
@@ -118,53 +119,57 @@ const AboutTheDataPage = ({
                     <h2 className="header">About These Statistics</h2>
                     <p className="sub-header">Agencies submit data monthly and/or quarterly to USAspending.gov. The table below shows information about the status and content of agency financial data submissions, and it will be updated as agencies publish/certify new submissions or republish/recertify existing submissions.</p>
                 </div>
-                <div className="table-controls">
-                    <Tabs
-                        active={activeTab}
-                        switchTab={handleSwitchTab}
-                        types={[
-                            {
-                                internal: 'submissions',
-                                label: "Statistics by Submission Period",
-                                labelContent: <TableTabLabel label="Statistics by Submission Period" />
-                            },
-                            {
-                                internal: 'publications',
-                                label: "Updates by Fiscal Year",
-                                labelContent: <TableTabLabel label="Updates by Fiscal Year" />
-                            }
-                        ]} />
-                    <TimeFilters
-                        submissionPeriods={submissionPeriods}
-                        latestFy={latestFy}
-                        latestPeriod={latestPeriod}
-                        activeTab={activeTab}
-                        onTimeFilterSelection={onTimeFilterSelection}
-                        selectedPeriod={selectedPeriod}
-                        selectedFy={selectedFy}
-                        urlPeriod={urlPeriod}
-                        urlFy={urlFy} />
-                </div>
-                <AgenciesContainer
-                    openModal={modalClick}
-                    activeTab={activeTab}
-                    selectedFy={selectedFy}
-                    selectedPeriod={selectedPeriod
-                        ? selectedPeriod.id
-                        : ''
-                    } />
-                <Note message={message} />
-                <AboutTheDataModal
-                    mounted={!!showModal.length}
-                    type={showModal}
-                    className={modalClassNames[showModal]}
-                    title={modalTitles(modalData?.type)[showModal]}
-                    agencyData={{
-                        ...modalData,
-                        fiscalYear: parseInt(selectedFy, 10),
-                        fiscalPeriod: parseInt(selectedPeriod?.id, 10) || 0
-                    }}
-                    closeModal={closeModal} />
+                <LoadingWrapper isLoading={!activeTab}>
```

Comment:
The more I think about it, is this loading wrapper hurting our SEO.  Should we directly be loading static data from our different components and letting those components render the dynamic data?

Reply:
Not sure exactly what you're asking, but I am asking a similar question. I do think we need to handle the loading state by section rather than by page.